src/sql/sqlops.d.ts

```diff
@@ -2342,7 +2342,8 @@ declare module 'sqlops' {
 
 	export namespace window {
 		/**
-		 * creates a dialog
+		 * NOTE: This method will be deprecated in a future release.
```

Comment:
You should use the js doc `@deprecated` tag for these comments: http://usejsdoc.org/tags-deprecated.html

Reply:
cool. thanks!